### PR TITLE
Bv 126/switch comp

### PR DESCRIPTION
**Pull request by Milena-Wasiak**

### CODE REVIEW

**Review by wojciech-nowakowski — APPROVED** [review]
dla mnie ok

File: src/components/DashboardTable/DashboardTableHeader-view.tsx [hunk 1]
```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import Legend from '../Legend';
 import { useDashboardTableStyles } from './DashboardTable-styles';
+
+import SwitchButtonComponent from '../SwitchButton';
```
Comment (dariuszsobkowicz) [inline src/components/DashboardTable/DashboardTableHeader-view.tsx]: why not `SwitchButton` ?

**Review by agatascalac — APPROVED** [review]
spoxik 👌

**Review by dariuszsobkowicz — APPROVED** [review]
looks great 🤦‍♂️

**Pull request merged**